modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/build.mustache

```diff
@@ -0,0 +1,139 @@
+{{#useAndroidMavenGradlePlugin}}
+group = '{{groupId}}'
+project.version = '{{artifactVersion}}'
+{{/useAndroidMavenGradlePlugin}}
+
+buildscript {
+
+    ext.kotlin_version = '1.5.10'
+
+    ext.swagger_annotations_version = "1.6.2"
+
+    ext.gson_version = "2.8.6"
+
+    ext.volley_version = "1.2.0"
+
+    ext.junit_version = "4.13.2"
+
+    ext.robolectric_version = "4.5.1"
+
+    ext.concurrent_unit_version = "0.4.6"
+
+    repositories {
+        mavenLocal()
+        google()
+        maven {
+            url 'https://dl.google.com/dl/android/maven2'
+        }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
```

Comment:
@alisters could you please reference the ext.kotlin_version here please?

Reply:
My bad, didn't pull the changes back into the mustache file. Fixed now